samples/betaFeatures.js

```diff
@@ -0,0 +1,540 @@
+      const chunks = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      const lastChunk = chunks[chunks.length - 1].words;
+      lastChunk.forEach(a => console.log(JSON.stringify(a, null, 4)));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_diarization_gcs]
+}
+
+function speechTranscribeMultiChannel(fileName) {
+  // [START speech_transcribe_multichannel]
+  //   Transcribe the given audio file asynchronously with
+  //     the selected model.
+
+  const fs = require('fs');
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const fileName = 'Local path to audio file, e.g. /path/to/audio.raw';
+
+  const config = {
+    encoding: `LINEAR16`,
+    languageCode: `en-US`,
+    audioChannelCount: 2,
+    enableSeparateRecognitionPerChannel: true,
+  };
+
+  const audio = {
+    content: fs.readFileSync(fileName).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(
+          result =>
+            ` Channel Tag: ` +
+            result.channelTag +
+            ` ` +
+            result.alternatives[0].transcript
+        )
+        .join('\n');
+      console.log(`Transcription: \n${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_multichannel]
+}
+
+function speechTranscribeMultilang(fileName) {
+  //[START speechTranscribeMultilang]
+
+  const fs = require('fs');
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const fileName = 'Local path to audio file, e.g. /path/to/audio.raw';
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+    audioChannelCount: 2,
+  };
+
+  const audio = {
+    content: fs.readFileSync(fileName).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+
+  //[END speech_transcribe_multilang]
+}
+
+function speechTranscribeMultilangGCS(gcsUri) {
+  // [START speech_transcribe_multilang_gcs]
```

Comment:
Is this snippet needed? _(see [this comment](https://github.com/googleapis/nodejs-speech/pull/118/files#r204939549) for full set of Python region tags)_